pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -478,11 +479,14 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 	})
 
 	Context("with VirtualMachineInstance spec", func() {
+		var vmi *v1.VirtualMachineInstance
+
+		BeforeEach(func() {
+			vmi = api.NewMinimalVMI("testvmi")
```

Comment:
Is it not necessary to set `vmi.Spec.Architecture` here?

Reply:
No only two tests require vmi.spec.arch remaining doesn't require arch as the tests are looking for valid and invalid hostname, subdomain name, audio devices etc thats why i didn't add in before each.